### ISSUE TITLE
Thunder Scans Paid Chapters Causing Issues

### ISSUE THREAD

**Issue by AgentX2006** [keiyoushi/tachiyomi-extensions-source#6144]
### Source information

Thunder Scans 1.4.35

### Source language

English

### Steps to reproduce

1. Add a series with multiple paid chapters to your library such as Growth-type Superhero (https://en-thunderscans.com/comics/growth-type-superhero/)

### Expected behavior

Either all chapters should be shown or preferably all paid chapters hidden (this preference due to the issue in the Other Details section below).

### Actual behavior

Only the latest paid chapter is shown followed by the latest free chapter. The ones in between are listed as missing.
![image](https://github.com/user-attachments/assets/4f04de65-4289-4d65-984f-473739c080ac)

### Mihon/Tachiyomi version

Tachiyomi Stable 0.15.3

### Android version

Android 14

### Other details

This also causes another issue in Tachiyomi when using Auto download while reading. When it goes to download a paid chapter it buggs the download queue and all downloads are stuck waiting on it until you manually clear the paid chapter that keeps failing from the queue.

This could be avoided by including a toggle option like some extensions have (Animated Clitched Comics is one example) which allows all paid chapters to be shown/hidden.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by Rstoroniomi** [comment]
你们都是怎么问问题的，为什么我找不到问题按钮呢，
How did you ask for problem？I cannot look for question button？

**Comment by Rstoroniomi** [comment]
请增加空白问题按钮。
pleaseadd blanket issue button.thanks

**Comment by mrtear** [comment]
probably related to #5908

https://github.com/keiyoushi/extensions-source/issues/5908#issuecomment-2466734304

**Comment by AwkwardPeak7** [comment]
Duplicate of #5908
